ex2_system/source/diagnostic/diagnostic.c

```diff
@@ -311,7 +316,7 @@ SAT_returnState start_diagnostic_daemon(void) {
 #endif
 
 #ifndef CHARON_IS_STUBBED
-    if (xTaskCreate((TaskFunction_t)charon_watchdog_daemon, "charon_watchdog_daemon", 2048, NULL, DIAGNOSTIC_TASK_PRIO,
+    if (xTaskCreate(charon_watchdog_daemon, "charon_watchdog_daemon", 2048, NULL, 3,

```

Comment:
Why the removal of the macro?

Reply:
Oh I didn't mean to include that - good catch. Was a debugging attempt